beagle_etl/copy_service/copy_service.py

```diff
@@ -8,23 +8,33 @@
 
 
 class CopyService(object):
-
     @staticmethod
     def copy(path_from, path_to):
-        logger.info('Copy path from {path_from} to {path_to}'.format(path_from=path_from,
-                                                                     path_to=path_to))
+        logger.info(
+            "Copy path from {path_from} to {path_to}".format(
+                path_from=path_from, path_to=path_to
+            )
+        )
         dirname = os.path.dirname(path_to)
         if not os.path.exists(dirname):
             os.makedirs(dirname, mode=settings.COPY_DIR_PERMISSION)
         copyfile(path_from, path_to)
         os.chmod(path_to, settings.COPY_FILE_PERMISSION)
 
+        for dirpath, dirnames, filenames in os.walk(path_to):
+            shutil.chown(dirpath, group=settings.COPY_GROUP_OWNERSHIP)
+            for filename in filenames:
+                shutil.chown(
```

Comment:
This will chown even `/juno/work/ci/` I think. Is that expected?

Reply:
If that's what it does, I will have to change it - it should only set permissions for files/dirs below that root dir